docs/assets/stylesheets/extra.css

```
@@ -1,19 +1,81 @@
+/* Inter — loaded by Material (mkdocs theme.font.text) */
+:root {
+  --ck-font-sans: "Inter", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)